src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -306,13 +306,18 @@ final class ListBuffer[A]
   def result: List[A] = toList
 
   /** Converts this buffer to a list. Takes constant time. The buffer is
-   *  copied lazily, the first time it is mutated.
+   *  copied lazily the first time it is mutated.
    */
   override def toList: List[A] = {
     exported = !isEmpty
     start
   }
 
+  // scala/bug#11869
+  override def toSeq: immutable.Seq[A] = toList
```

Comment:
Should be binary compatible if you make the return type `scala.collection.Seq`

Reply:
Which, in 2.12-land, is just `Seq`.

Reply:
Yep, thanks. I'm going back and forth between 2.13 and 2.12 enough to get confused.